content/reference-ledger-format.md

```diff
@@ -239,6 +240,64 @@ The lower 64 bits of an Offer Directory's index represent the TakerPays amount d
 * The page number of this node. (Since 0 is the root DirectoryNode, this value is an integer 1 or higher.)
 
 
+## Escrow
+[[Source]<br>](https://github.com/ripple/rippled/blob/c6b6d82a754fe449cc533e18659df483c10a5c98/src/ripple/protocol/impl/LedgerFormats.cpp#L90-L101 "Source")
```

Comment:
I know this breaks with the convention of other "source" links, but @nbougalis suggested that linking to https://github.com/ripple/rippled/blob/develop/src/ripple/app/tx/impl/Escrow.cpp might make a little more sense than that specific commit.